src/Components/Authentication/Desktop/ModalManager.tsx

```diff
@@ -86,11 +88,11 @@ export class ModalManager extends Component<
           handleSubmit={handleSubmit}
           onFacebookLogin={() =>
             (window.location.href =
-              submitUrls.facebook + "?redirect-to=" + redirectTo)
+              submitUrls.facebook + "?redirect-to=" + options.redirectTo)
```

Comment:
No action necessary now, but note that we'll need to add more params for analytics later on --

Reply:
👌🏾 yup